frontend/src/components/ScheduledEventsTableRow/utils/dateConversion.ts

```diff
@@ -0,0 +1,24 @@
+import { ScheduledEventsTableRowProps } from '../ScheduledEventsTableRow';
+
+export const deriveTimeFromStartDate = (props: ScheduledEventsTableRowProps): string => {
+  const dt = new Date();
+  dt.setTime(Date.parse(props.start_date));
+  let hours = dt.getUTCHours();
+  const AmOrPm = hours >= 12 ? 'PM' : 'AM';
+  hours = hours % 12 || 12;
+  const minutes = dt.getUTCMinutes();
+  const derivedTime = hours + ':' + (minutes < 10 ? '0' + minutes : minutes) + ' ' + AmOrPm;
+
+  return derivedTime;
+};
+
+export const deriveDateFromStartDate = (props: ScheduledEventsTableRowProps): string => {
```

Comment:
`deriveDateFromDate` ... use date as a parameter not all the props.

FYI (don't need it now since u already have this): https://momentjs.com/ ... up to you if you want to replace all this with moment